### PR TITLE
Feature stickers

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

File: lib/copyResources.js [hunk 1]
```diff
@@ -126,7 +205,7 @@ exports.copySplash = function (api, app, config) {
       var jvmExec = api.jvmtools.exec;
       return Promise
         .fromNode(jvmExec.bind(api.jvmtools, {
-          tool: 'splasher',
+          tool: 'splasher',
```
Comment (rampr) [inline lib/copyResources.js]: extra trailing space

**Review by rampr — APPROVED** [review]

File: build.js [hunk 2]
```diff
@@ -247,11 +247,17 @@ function buildXcodeProject(api, app, config) {
...
+
       var configPlist = updatePlist.get(path.join(config.xcodeProjectPath, 'resources', 'config.plist'));
       var entitlements = updatePlist.get(path.join(config.xcodeProjectPath, 'TeaLeafIOS.entitlements'));
       updateInfoPlist(app, config, infoPlist);
       updateConfigPlist(app, config, configPlist);
 
+      var stickersPlist = updatePlist.getInfoPlist(config.xcodeProjectPath + '/stickers');
+      var raw = stickersPlist.getRaw();
+      raw.CFBundleDisplayName = app.manifest.title || "";
+      raw.CFBundleIdentifier = config.bundleID + '.sticker';
```
Comment (bijoshtj) [inline build.js:259]: these should be done only if sticker is enabled from manifest?

File: lib/copyResources.js (hunk 1 above)
Comment (vivekb91) [reply lib/copyResources.js]: will remove

File: tealeaf/TeaLeafIOS.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -187,8 +187,43 @@
...
+			isa = PBXBuildFile;
+			fileRef = E25AB2641F97557200069969 /* Stickers.xcassets */;
+		};
+		E25AB2691F97557200069969 /* stickers.appex in Embed App Extensions */ = {
+			isa = PBXBuildFile;
+			fileRef = E25AB2621F97557100069969 /* stickers.appex */;
+			settings = {
+				ATTRIBUTES = (RemoveHeadersOnCopy, );
+				};
+			};
```
Comment (bijoshtj) [inline tealeaf/TeaLeafIOS.xcodeproj/project.pbxproj:200]: what these changes for??

**Review by Javasamurai — APPROVED** [review]

File: build.js (hunk 2 above)
Comment (rampr) [reply build.js:259]: Vivek spoke with me and he mentioned this will done as the second step.

**Review by bijoshtj — APPROVED** [review]

**Pull request merged**